File: _move.py
import shutil
import os

src_dir = r"d:\Projects\tablet-issuance\issuance-frontend\app\api"
dst_dir = r"d:\Projects\tablet-issuance\issuance-frontend\src\app\api"
app_dir = r"d:\Projects\tablet-issuance\issuance-frontend\app"

try:
    if os.path.exists(dst_dir):
        shutil.rmtree(dst_dir)
    shutil.copytree(src_dir, dst_dir)
    print("Copied successfully.")
    try:
        shutil.rmtree(app_dir)
        print("Removed original app dir.")
    except Exception as e:
        print("Removed failed:", e)
        # Try renaming it instead
        os.rename(app_dir, r"d:\Projects\tablet-issuance\issuance-frontend\app_hidden")
        print("Renamed instead.")
except Exception as e:
    print("Error:", e)

File: _move.js
const fs = require('fs');
const path = require('path');

const srcDir = 'd:\\Projects\\tablet-issuance\\issuance-frontend\\app\\api';
const destDir = 'd:\\Projects\\tablet-issuance\\issuance-frontend\\src\\app\\api';
const appDir = 'd:\\Projects\\tablet-issuance\\issuance-frontend\\app';

try {
    fs.cpSync(srcDir, destDir, { recursive: true });
    console.log('Copied successfully');
    fs.renameSync(appDir, 'd:\\Projects\\tablet-issuance\\issuance-frontend\\app_hidden');
    console.log('Renamed app successfully');
} catch (e) {
    console.error('Error:', e.message);
}
